release/tasks/get-packages.ts

```diff
@@ -0,0 +1,116 @@
+import * as path from 'path';
+
+import globby from 'globby';
+import latest from 'latest-version';
+
+import { Context, Reference } from '../@types/custom';
+import { debug } from '../lib/utils';
+
+/** Loads all the references of a `tsconfig.json` file to resolve dependencies. */
+const loadReferences = (route: string): string[] => {
+    try {
+        const tsconfig = require(path.resolve(process.cwd(), route));
+
+        const references = tsconfig.references ?
+            tsconfig.references.map((reference: Reference) => {
+                return reference.path;
+            }) :
+            [];
+
+        return references;
+    } catch (e) {
+        return [];
+    }
+};
+
+const shouldBeIgnored = (destination: string, ignoredPackages: string[]) => {
+    return ignoredPackages.some((ignoredPkg) => {
+        return destination.includes(ignoredPkg);
+    });
+};
+
+/**
+ * Populate the context with all the valid packages to process. Packages
+ * listed in `ignoredPackages` or with the field `"private": true`
+ * in their `package.json` will have the property `ignore: true`
+ */
+export const getPackages = (ignoredPackages: string[]) => {
+    return async (ctx: Context) => {
+        debug(`Loading packages`);
+        ctx.packages = new Map();
+
+        const ignored = ignoredPackages.map((pkg) => {
+            return path.join(process.cwd(), 'packages', pkg);
+        });
+
+        const pkgPaths = await globby([
+            'packages/**/*/package.json',
```

Comment:
Do we support nested packages generally, or would `'packages/*/package.json'` suffice here?

Reply:
I don't know what the future holds and it was easy enough to have it this way and just negate the `fixtures`